arelle/PluginManager.py

```diff
@@ -297,7 +328,15 @@ def moduleModuleInfo(moduleURL, reload=False, parentImportsSubtree=False):
                             if _importModuleInfo:
                                 _importModuleInfo["isImported"] = True
                                 imports.append(_importModuleInfo)
-                        moduleInfo["imports"] =  imports
+                        moduleInfo["imports"] = imports
+                        if entryPoint:
+                            moduleInfo["moduleURL"] = moduleFilename  # pip-installed plugins need absolute filepath
+                            if not moduleInfo.get("version"):
+                                moduleInfo["version"] = entryPoint.dist.version  # If no explicit version, retrieve from entry point
```

Comment:
I noticed `ixbrl-viewer` was showing as version 0.1, so I took the opportunity to update the version string with the pip package version if one was not found in the `__pluginInfo__` object.

Reply:
I think this should be flipped. Use the entry point version if it exists, otherwise use the module info version

Reply:
It makes sense to me to leave it up to the plugin to decide which to use. If the plugin maintainer intends for the package version to be used, it wouldn't make sense for them to be providing a hardcoded version. On the other hand, if they *don't* want the package version used, this allows them to override it.